tests/src/test/scala/quasiquotes/ErrorSuite.scala

```diff
@@ -297,9 +297,11 @@ class ErrorSuite extends FunSuite {
       val tpes = List(q"T")
       q"expr: ..$tpes"
     """) === """
-      |<macro>:5: identifier expected but ellipsis found
+      |<macro>:5: type mismatch when unquoting;
+      | found   : List[scala.meta.Term.Name]
+      | required: scala.collection.immutable.Seq[scala.meta.Type]
```

Comment:
For list of types it says that wants a single type. Agree, so previous message is lying, and need to be changed.